### PR TITLE
add privateID username check

### PR DESCRIPTION
**Pull request by mchangrh**
- add privateID check to setUsername
- bump AGPL to package-lock
- [x] I agree to license my contribution under AGPL-3.0-only and LGPL-3.0

### CODE REVIEW

**Comment by ajayyy** [comment]
cool

**Comment by ajayyy** [comment]
oh tests failed rip

**Comment by mchangrh** [comment]
oh

**Comment by mchangrh** [comment]
try running the postgres test again? beforeAll setUsername should be just sql

**Comment by ajayyy** [comment]
it failed in dependabot too

File: test/cases/setUsername.ts [hunk 1]
```diff
@@ -88,6 +98,10 @@ describe("setUsername", () => {
         await addUsername(getHash(user05PrivateUserID), username05, 0);
         await addUsername(getHash(user06PrivateUserID), username06, 0);
         await addUsername(getHash(user07PrivateUserID), username07, 1);
+        await addUsername(getHash(user07PrivateUserID), username07, 0);
```
Comment (ajayyy) [inline test/cases/setUsername.ts:101]: Duplicate lines

**Pull request merged**